test/test_cuda.py

```diff
@@ -2339,6 +2339,7 @@ def plan_cache_max_size(n, device=None):
                             self.assertEqual(torch.backends.cuda.cufft_plan_cache.max_size, 10)  # default is cuda:0
                         self.assertEqual(torch.backends.cuda.cufft_plan_cache.max_size, 11)  # default is cuda:1
 
+    @skipIfRocm
```

Comment:
Do you know why do these tests fail?

Reply:
not entirely clear to me. I suspect that the use of librosa in stft may be an issue here.

Reply:
I see. In this case could you add a comment here to indicate this test passes in py2 but fails in py3?